skt/runner.py

```diff
@@ -649,9 +649,11 @@ def __jobsubmit(self, xml):
             args += ["--job-owner=%s" % self.jobowner]
 
         args += ["-"]
-
-        stdout, _, _ = safe_popen(args, stdin_data=xml, stdin=subprocess.PIPE,
-                                  stdout=subprocess.PIPE)
+        err_strings = ['connection to beaker.engineering.redhat.com failed',
```

Comment:
Nitpick: Maybe use `"` for this string, too, so it looks like the other one?

Reply:
sure